hawc_hal/region_of_interest/healpix_map_roi.py

```diff
@@ -31,6 +31,8 @@ def __init__(self, model_radius, roimap=None, roifile=None, threshold=0.5, *args
 
         :param model_radius: radius of the model cone. Either an astropy.Quantity instance, or a float, in which case it
         is assumed to be the radius in degrees
+        :param data_radius: radius used for displaying maps. Either an astropy.Quantity instance, or a float, in which case it
```

Comment:
Digression: The GitHub interface only allows me to comment on changed lines +/- 3 lines? This comments is for some code 4 and 9 lines above this one...

The `roi = HealpixMapROI(...)` examples above should be updated with the new keyword.

Reply:
Done :)

Reply:
@colasri do you agree with these changes?